extensions/wikia/PageHeader/styles/index.scss

```diff
@@ -119,6 +119,10 @@
 		line-height: 1;
 		max-width: 100%;
 		word-wrap: break-word;
+
+		i {
```

Comment:
is the italic style only one that we should support? did you ask stanley/tor/julia about this?